src/main/java/com/boot/bankservice/service/AccountService.java

```diff
@@ -0,0 +1,16 @@
+package com.boot.bankservice.service;
+
+import com.boot.bankservice.model.Account;
+import java.util.Optional;
+
+public interface AccountService {
+    Account add(Account account);
+
+    Optional<Account> get(Long id);
+
+    Account update(Account account);
```

Comment:
"You can have a single save() method on the service layer for both ‘create’ and ‘update’ operations."

Reply:
yes, I know this, but I prefer have two methods)